tests/readyflag/tst_readyflag_local.qml

```diff
@@ -0,0 +1,48 @@
+import QtTest 1.2
+import tests.readyflag  1.0
+
+TestCase {
+
+    ReadyFlagInterface{
+        id: iReady
+    }
+
+    function test_readyInt() {
+
+        compare(iReady.readyFlags.intProperty,false)
+        tryVerify(function() {
+            iReady.intProperty = 10;
+            return true;
```

Comment:
tryVerify is useless, if you always return true. I guess what you want is:
api.intProperty = 10
tryVerify(api.readyFlags.intProperty)
but since it is all synchronous it is probably not needed.